ContactMechanics/Optimization/ConstrainedConjugateGradients.py

```diff
@@ -41,11 +41,13 @@
 def constrained_conjugate_gradients(substrate, topography, hardness=None,
                                     external_force=None, offset=None,
                                     disp0=None,
```

Comment:
Can we rename `disp0` to `initial_displacements` and `forces0` to `initial_forces`?

Reply:
Agreed. This is the perfect moment to do that change since it is an API change.

Reply:
Please also update the change log to reflect this API change